codecs/h264_packet.go

```diff
@@ -55,15 +94,30 @@ func emitNalus(nals []byte, emit func([]byte)) {
 	if nextIndStart == -1 {
 		emit(nals)
 	} else {
+		firstNalStart := true
 		for nextIndStart != -1 {
 			prevStart := nextIndStart + nextIndLen
-			nextIndStart, nextIndLen = nextInd(nals, prevStart)
+			// Check for SEI subtitle NAL unit (We should expect it in first NAL)
+			if firstNalStart {
+				isSubtitle, endSubtitlePos := h264SubtitleSeiAttemptParse(nals, nextIndStart+nextIndLen)
+				if isSubtitle {
+					/* Subtitle NAL may contain nal unit start code (Does not escaped with emulation prevention byte),
+					so we ensure the next NAL start code index will be searched for after the end of SEI subtitle nal unit */
```

Comment:
```suggestion
					/* subtitles are being encoded as SEI nalus.
					    each nalu has a start code which must be escaped.
					    since we don't escape our subtitle SEI start code as part of our encoding, we skip it until the next nalu. */
```

Reply:
resolved?